Project_source/app/src/main/java/android/clicker/school_live_simulator/StudyCourseState.kt

```diff
@@ -0,0 +1,14 @@
+package android.clicker.school_live_simulator
+
+abstract class StudyCourseState: TimerObserver {
+    abstract val price: Int
```

Comment:
Price field should have protected access type.

Reply:
Fixed